src/Tools/plugins/widget/customwidgets.cpp

```diff
@@ -1248,27 +1232,6 @@ void QuantitySpinBox::closeEvent(QCloseEvent* event)
 
 bool QuantitySpinBox::event(QEvent* event)
 {
-    // issue #0004059: Tooltips for Gui::QuantitySpinBox not showing
-    // Here we must not try to show the tooltip of the icon label
-    // because it would override a custom tooltip set to this widget.
-    //
-    // We could also check if the text of this tooltip is empty but
-    // it will fail in cases where the widget is embedded into the
-    // property editor and the corresponding item has set a tooltip.
-    // Instead of showing the item's tooltip it will again show the
-    // tooltip of the icon label.
-#if 0
-    if (event->type() == QEvent::ToolTip) {
-        if (isBound() && getExpression() && lineEdit()->isReadOnly()) {
-            QHelpEvent * helpEvent = static_cast<QHelpEvent*>(event);
-
-            QToolTip::showText( helpEvent->globalPos(), Base::Tools::fromStdString(getExpression()->toString()), this);
-            event->accept();
-            return true;
-        }
-    }
-#endif
-
     return QAbstractSpinBox::event(event);
 }

```

Comment:
Now when you've removed this old workaround, you can safely remove this whole function.